src/SprykerSdk/Zed/AppSdk/Communication/Console/AppTranslationBuilderConsole.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AppSdk\Communication\Console;
+
+use Generated\Shared\Transfer\AppTranslationRequestTransfer;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\Question;
+
+/**
+ * @method \SprykerSdk\Zed\AppSdk\Business\AppSdkFacadeInterface getFacade()
+ */
+class AppTranslationBuilderConsole extends AbstractConsole
+{
+    /**
+     * @var array
+     */
+    protected $translations = [];
```

Comment:
```suggestion
    protected array $translations = [];
```
Please also add a proper doc block for the array, I guess it's `array<string>`.